### ISSUE TITLE
Clean up config class

### ISSUE THREAD

**Issue by ShishKabab** [oceanprotocol/squid-py#233]
The Config class is now an ini loader and has access to web3. Since config has nothing to do with web3, this link should be removed. Also, the loading should be separated from the Config class itself (a Config is not a loader, but can maybe use/have a loader), even if for convenience a file path string may still be passed in.

**Comment by MarcusJones** [comment]
Ref #226, #63 

Re: loading should be separated - I don't mind this pattern, to have a Data class with a self.load() is something quite common. We a just sub-classing the builtin ConfigParser, which does exactly that. Data Scientists are especially comfortable with i.e. [.from_dict()](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.from_dict.html), and .from_csv() style Data classes. What do you propose more concretely?

**Comment by ShishKabab** [comment]
Yeah, it's quite common, but not a very clean pattern. When not following the single responsibility principles, it becomes very easy to grow classes that load config, represent config, calculate derived config, signal other components config has changed, etc. I'd propose just having a few functions, like `load_config_from_ini()`, `load_config_from_json()`, etc. that return a Config object. Config itself would just take a dict, because that's not specific to any file format or source.

Also, removing web3 from this class would be a higher prio.
